python/samples/documentation_examples/plugin.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import sys
+
+from service_configurator import add_ai_service
+
+import semantic_kernel as sk
+
+if sys.version_info >= (3, 9):
+    from typing import Annotated
+else:
+    from typing_extensions import Annotated
+
+from semantic_kernel.functions.kernel_function_decorator import kernel_function
+
+
+async def main():
+    # Initialize the kernel
+    kernel = sk.Kernel()
+
+    # Add the service to the kernel
+    # use_chat: True to use chat completion, False to use text completion
+    kernel = add_ai_service(kernel=kernel, use_chat=True)
+
+    light_plugin = kernel.import_plugin_from_object(
+        LightPlugin(),
+        plugin_name="LightPlugin",
+    )
+
+    # Kernel Arguments are passed in as kwargs.
+    result = await kernel.invoke(light_plugin["get_state"])
+    print(f"The light is: {result}")
+
+    print("Changing the light's state...")
+
+    result = await kernel.invoke(light_plugin["change_state"], new_state=True)
+    print(f"The light is: {result}")
+
+
+class LightPlugin:
```

Comment:
maybe define this first before the main, I was confused for a sec where this came from.